src/pages/explore/Explore.jsx

```diff
@@ -1,15 +1,41 @@
 import Post from "components/Post";
-import TextEditor from "components/TextEditor";
+import { useState } from "react";
 import { useSelector } from "react-redux";
 
 export default function Explore() {
   const { allPosts } = useSelector((state) => state.allPosts);
+  const [sortBy, setSortBy] = useState("Latest");
+
+  const sortHandler = () => {
```

Comment:
This can be moved into services.

Reply:
Thank you, I plan to refactor it post adding all the features.